app/controllers/accounts_controller.rb

```diff
@@ -78,12 +79,15 @@ def no_replies_scope
     Status.without_replies
   end
 
+  def hashtag_scope
+    Status.tagged_with(Tag.find_by(name: params[:tag].downcase)&.id)
```

Comment:
Not sure what the project's preference is, but I've been seeing a lot of people move from `#[]` to `#fetch()` since it tells the reader a lot more about the intent. For example, in this method we *EXPECT* `:tag` to be a key, thus `params.fetch(:tag)`, but if we didn't `params.fetch(:tag, nil)` and of course we get all the niceness of `#fetch`'s other default api: `fetch(:key) {dynamicvalue}`.

Either way, seems like what you want here could also be expressed as: `Status.tagged_with(Tag.where(name: params[:tag]).pluck(:id).first)`, though if `tagged_with` allows an array you can just drop `.first`.